Percolator/Percolator.toolbox/Percolator___PercolatorModel.launch

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
do we need this file?

Reply:
this file is the TLC model file for tla+ toolbox, it is necessary if users would like to test our spec against some model, since it would be hard to describe our testing model precisely in natural language.

`elasticsearch-formal-models` also includes this file, see https://github.com/elastic/elasticsearch-formal-models/blob/master/data/tla/replication.toolbox/replication___model.launch

But other two files are not needed I think, as when users are importing `Percolator.tla`, these two files will be generated automatically.